spec/support/capybara.rb

```diff
@@ -1,28 +1,19 @@
 require "capybara/rspec"
-begin
-  require "webdrivers"
-rescue LoadError
-  nil
-end
-
-view_mode = (ENV.fetch("VIEW", false) == "true") ? :chrome : :headless_chrome
-args = %w[disable-dev-shm-usage no-sandbox window-size=1280x1280]
-args <<= "headless" unless view_mode == :chrome
+require "capybara/cuprite"
 
-Capybara.register_driver view_mode do |app|
-  Capybara::Selenium::Driver.new(
+Capybara.register_driver(:cuprite) do |app|
+  Capybara::Cuprite::Driver.new(
     app,
-    browser: :chrome,
-    options: Selenium::WebDriver::Options.chrome(
-      args: args
-    )
+    window_size: [1200, 800],
+    process_timeout: 20,
+    timeout: 10
```

Comment:
Probably useful to keep supporting the `VIEW` setting. I find it's pretty handy for the odd bit of debugging
```suggestion
    timeout: 10,
    headless: ENV.fetch("VIEW", nil).blank?
```

Reply:
cuprite supports screenshots but unfortunately does not seem to support non-headless mode